content/zotero_intro_to.md

```diff
@@ -359,7 +359,7 @@ We will focus on some most useful features as you are starting your library, tha
 * searches -> quick vs. advanced, immediate vs. saved with dynamic update,.
 * notes.
 
-**/!\\** In Zotero, the icon associated with a **collection** is a folder but collections are more like **music playlists** (think of iTunes, Spotify, etc.) than folders in your computer filesystem.
+**⚠️** In Zotero, the icon associated with a **collection** is a folder but collections are more like **music playlists** (think of iTunes, Spotify, etc.) than folders in your computer filesystem.
```

Comment:
to be left textual :point_right: find an Atom package supporting emoji when converting to PDF

Reply:
Ok